src/tests/ftest/security/pool_security_groups.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+'''
+
+import os
+import grp
+import pool_security_test_base as poolSec
+from pool_security_test_base import PoolSecurityTestBase
+
+class DaosRunPoolSecurityTest(PoolSecurityTestBase):
+    """Test daos_pool acl for primary and secondary groups.
+
+    :avocado: recursive
+    """
+    # pylint: disable=too-many-ancestors
+
+    def test_daos_pool_acl_groups(self):
+        '''
+        Epic:
+            DAOS-1961: Testing related to DAOS security features.
+        Testcase description:
+            DAOS-2951: Pool ACE/ACL identities verification.
+            DAOS-2955: Primary group user ACL verification.
+            DAOS-2957: Secondary group user ACL verification.
+            DAOS-3546: Test ACL access when user/group management
+                       isn't synced between client and server nodes.
+            DAOS-2961: DAOS dmg pool overwrite-acl verification
+            DAOS-2962: DAOS dmg pool deete-acl verification
```

Comment:
Didn't actually see anything about delete in here

Reply:
Thanks for the feedback.
I have added a new test step (6) to verify the acl ace entry add and delete.